db.go

```diff
@@ -52,11 +52,44 @@ func (net *Network) GetName() string {
 	return net.Addr
 }
 
+type MessageFilter int
+
+const (
+	FilterNone      MessageFilter = 0
+	FilterHighlight               = 1
+	FilterMessage                 = 2
+)
+
+func parseFilter(filter string) (MessageFilter, error) {
+	switch filter {
+	case "none":
+		return FilterNone, nil
+	case "highlight":
+		return FilterHighlight, nil
+	case "message":
+		return FilterMessage, nil
+	}
+	return 0, fmt.Errorf("unknown filter: %q", filter)
+}
+
 type Channel struct {
 	ID       int64
 	Name     string
 	Key      string
 	Detached bool
+
+	DetachTimeout int64 // seconds
```

Comment:
Even if the DB stores this as an int64 in seconds, maybe it would be better to make this a `time.Duration` field to make it easier to manipulate elsewhere?